test_name: "cartesian"
group_name: "cartesian_arm"
base_link: "world"
last_link: "end_effector"
dt_replan_restricted: 0.050
dt_replan_relaxed: 0.100
trj_execution_thread_frequency: 100
collision_checker_thread_frequency: 30
spawn_objs: true

start_configuration: [0.0, 0.0, 0.0]
stop_configuration: [0.8, 0.8, 0.8]
k_freq: 1
init_test: 0
end_test: 100
opt_path: true
